lmdeploy/pytorch/configurations/deepseek_v2.py

```diff
@@ -23,6 +26,14 @@ def build(cls, hf_config, model_path: str = None, **kwargs):
         tp = kwargs.get('tp', 1)
         # update num_kv_heads for tp mode
         num_key_value_heads = cls.update_num_kv_heads(hf_config, tp, num_key_value_heads)
+        # use flash_mla by default if it is installed
+        use_flash_mla = False
```

Comment:
There is flash_mla check everywhere.
I think we just need to check it when building nn.Attention module.

Reply:
Do you mean we use `use_flash_mla=True` here? There is no CLI option for users to use. So, I just set it here.

Reply:
Could we dispatch implement here https://github.com/InternLM/lmdeploy/blob/8549698b899f6bfd1bc59e85864de2cfab1cce11/lmdeploy/pytorch/backends/cuda/attention.py#L203